pycbc/frame/frame.py

```diff
@@ -906,16 +906,14 @@ def indices_of_flag(self, start_time, duration, times, padding=0):
         duration: int
             Number of seconds to check.
         padding: float
-            Number of seconds to add around flag inactive times to be considered
-        inactive as well.
+            Amount of time in seconds to flag around glitchy times
```

Comment:
glitchy doesnt feel right - maybe `Amount of time in seconds to extend around idq-flagged times` or something similar?

Reply:
(This preference may be due to having watched too much [Go-Jetters](https://gojetters.fandom.com/wiki/Grandmaster_Glitch))